src/Components/App/App.js

```diff
@@ -1,11 +1,23 @@
 import React, { Component } from 'react';
+import {useSelector, useDispatch} from 'react-redux';
+import { getInitialBooks } from '../../actions';
 import Search from '../Search/Search';
+import { connect } from 'react-redux';
+import { getBooks } from '../../apiCalls.js';
 import './App.css';
 
 class App extends Component {
-  constructor() {
-    super();
-    this.state = {}
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidMount = async () => {
+  const bookList = await getBooks()
+  const allBooks = bookList.results.books
+  this.props.getInitialBooks(allBooks)
+  // .catch(error => console.log('error', error))
+  // refactor to set user message into state depending on where state will be stored
+
```

Comment:
Where should this fetch and action live? Here on App or on the component where it will be used (BookContainer)?

Reply:
I think App is an appropriate place for now but I agree as we get our next components going, but I agree BookContainer may be a more appropriate place.